GraphsV2/Heatmap/default.config.js

```diff
@@ -0,0 +1,25 @@
+import { theme } from "../../theme"
+
+export const properties = {
+    stroke: {
+        color: "grey",
+        width: "0.5px"
+    },
+    legend: {
+        show: false
+    },
+    mapColors: {
+        'InSla': theme.palette.greenColor,
```

Comment:
these colors should not be defined here, they are very specific to the application.